pajbot/modules/basic/stream_update.py

```diff
@@ -70,7 +70,7 @@ def generic_update(self, bot: Bot, source, message: str, field: str, extra_args:
             return
 
         log_msg = f'{source} updated the {field} to "{message}"'
-        bot.say(log_msg)
+        bot.safe_say(log_msg)
```

Comment:
No need to run safe_say on moderator-only commands.
The risks we mitigate normally with the change from `say` to `safe_say` is if users have banphrased usernames or are using the bot to say banphrased messages. I believe moderators can be trusted to use moderator commands not to abuse banphrases (they could just remove banphrases), so the potential confusion if a game name is banphrased and no response is given is not worth it.